packages/cli/src/commands/help.ts

```diff
@@ -142,7 +142,12 @@ export function outputArrayToString(outputArray: string[]) {
  * @returns
  */
 export function buildCommandSynopsisLine(command: Command) {
-  const line: string[] = [LOGO, chalk.bold(NAME), chalk.bold(command.name)];
+  const line: string[] = [
+    INDENT,
```

Comment:
Because I don't think we should indent the description (see below), I don't think we should indent the command.